bindings/python/examples/missing_polygon.py

```diff
@@ -0,0 +1,39 @@
+from pymanifold import CrossSection, FillRule
+
+
+def run():
+    # create cross-sections
+    cross_section = \
+        CrossSection(
+            [[
+                (-176.5416564941429, 62.267156291299955), (-177.11900294911175, 63.267144940124126),
+                (-177.11900294910944, 62.26716470718384), (-176.5416564941406, 62.26715569064245),
+                (-176.54165614735393, 61.26716470718384), (-175.9643096923828, 61.26715985212647),
+                (-175.96430934559382, 60.26716470718384), (-175.38696566496216, 60.26717407055541),
+                (-175.386962543836, 59.26716470718384), (-174.80960083007815, 59.26714979522481),
+                (-174.8096004832914, 58.26716470718384), (-174.23225470415798, 58.26715427684856),
+                (-175.01954376826416, 59.63078409350792), (-176.5416564941429, 62.267156291299955),
+            ]]
+        )
+    test_cross = \
+        CrossSection(
+            [[
+                (-174.70, 58.40),
+                (-174.50, 58.40),
+                (-174.50, 58.60),
+                (-174.70, 58.60),
+                (-174.70, 58.40),
+            ]]
+        )
+
+    diff = test_cross - cross_section
+    if diff.area() != 0:
+        print(f"Cross-section is missing a part of the polygon! "
```

Comment:
Why do you expect this area to be zero?

Reply:
I expect there to be overlap, so I want it to be greater than zero.

Reply:
That's not what this if condition is saying. Also, even if there is overlap between these polygons, I don't believe they have equal area, so I'm not sure where the `> 0` expectation is coming from.

Reply:
We do a diff between the test square and the polygon under test to check for overlap (there should be a full overlap). If the difference is small (I changed it to eps now), the polygon is overlapping the test rectangle. But we get zero overlap in my test case.

Reply:
The overlap is the intersection, `^`, not the difference.